<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>motor Interface 1.1</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../serial_moter/motor.ico</normaloff>../serial_moter/motor.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_console">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="title">
       <string>Motor Control :</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_state">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="text">
            <string>Motor States : </string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_MotorAxis">
               <property name="text">
                <string>电机选择</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="MotorAxisBox">
               <item>
                <property name="text">
                 <string>X轴</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Y轴</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Z轴</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="button_Start">
               <property name="text">
                <string>启动</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>icone/start green.png</normaloff>icone/start green.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_Pause">
               <property name="text">
                <string>暂停</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>icone/Pause Normal Red.png</normaloff>icone/Pause Normal Red.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="button_Home">
               <property name="text">
                <string>复位</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>icone/house.png</normaloff>icone/house.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_Stop">
               <property name="text">
                <string>急停</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>icone/stop.png</normaloff>icone/stop.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="button_Speed">
               <property name="text">
                <string>设置速度</string>
               </property>
               <property name="icon">
                <iconset resource="icone.qrc">
                 <normaloff>:/iconeMotor/icone/route.png</normaloff>:/iconeMotor/icone/route.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="box_Speed"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="button_Forward">
               <property name="text">
                <string>前进</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>icone/forward.png</normaloff>icone/forward.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_Backward">
               <property name="text">
                <string>后退</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>icone/backward.png</normaloff>icone/backward.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_Console">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="text">
            <string>Console :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Console" name="console_Window">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>10000000</width>
             <height>10000000</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>533</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>main</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionClearConsole"/>
    <addaction name="actionCurve"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuConnect"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConfigure"/>
   <addaction name="actionCurve"/>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset>
     <normaloff>icone/usb.png</normaloff>icone/usb.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset>
     <normaloff>icone/disconnect.png</normaloff>icone/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>icone/remove.png</normaloff>icone/remove.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset>
     <normaloff>icone/settings.png</normaloff>icone/settings.png</iconset>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>icone/question.png</normaloff>icone/question.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset>
     <normaloff>icone/Qt_logo_2016.svg.png</normaloff>icone/Qt_logo_2016.svg.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionClearConsole">
   <property name="icon">
    <iconset>
     <normaloff>icone/broom.png</normaloff>icone/broom.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionCurve">
   <property name="icon">
    <iconset>
     <normaloff>icone/curve.png</normaloff>icone/curve.png</iconset>
   </property>
   <property name="text">
    <string>Curve</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Console</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">console.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icone.qrc"/>
 </resources>
 <connections/>
</ui>
